crates/evm/src/test_utils.rs

```diff
@@ -60,6 +61,10 @@ impl<DB> Executor<DB> for MockExecutorProvider {
             gas_used: 0,
         })
     }
+
+    fn state_ref(&self) -> &State<DB> {
+        todo!()
+    }
```

Comment:
this now bakes in the state into the executor interface, I don't want to do this